kubetest/plugin.py

```diff
@@ -241,7 +241,8 @@ def pytest_runtest_teardown(item):
     See Also:
         https://docs.pytest.org/en/latest/reference.html#_pytest.hookspec.pytest_runtest_teardown
     """
-    manager.teardown(item.nodeid)
+    if hasattr(item.config, 'kube_config'):
```

Comment:
The logic behind this small addition is that if `kube-config` wasn't supplied, no resource was created so there is no need to destroy anything. If anyone has an idea about a different place in the code where I can handle the scenario described in the PR description, please share it.